step_definitions/GeolocationSteps.ts

```
@@ -0,0 +1,23 @@
+	}: {
+		latitude: number
+		longitude: number
+		accuracy: number
+	}) {
+		I.usePlaywrightTo('set permissions', async ({ page }) => {
+			const context = page.context()
+			await context.grantPermissions(['geolocation'])
+			await context.setGeolocation({ latitude, longitude, accuracy })
+		})
+	}
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)